docs/sources/vendor/a10networks/vthunder.md

```diff
@@ -0,0 +1,30 @@
+# a10networks vthunder
+
+## Key facts
+
+* MSG Format based filter
+* Legacy BSD Format default port 514
+
+## Links
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| Splunk Add-on  | None                                                                                                    |
```

Comment:
Not sure if community-owned add-ons count but there are some addons made by A10 networks that can be checked against regarding the sourcetype: https://splunkbase.splunk.com/apps?keyword=thunder